imports/core/api/players/methods.js

```diff
@@ -76,13 +78,22 @@ export const createPlayer = new ValidatedMethod({
     let lobbyPool = lobbies.filter(
       l => l.availableCount > l.queuedPlayerIds.length
     );
+
     // If no lobbies still have "availability", just fill any lobby
     if (lobbyPool.length === 0) {
       lobbyPool = lobbies;
```

Comment:
@amaatouq This is where we're deciding to overbook, you can add a filter here: `lobbies.filter(l => l.availableCount > 1)`. I think that should do the trick.